recipes/linux.rb

```diff
@@ -19,7 +19,7 @@
 #
 
 # Install Nexpose Pre-Reqs
-package 'screen'
+package 'screen' unless node['platform_family'] == 'rhel' # rhel requires CD/ISO to be mounted to install screen
```

Comment:
@rhass Agreed. I think `ignore_failure true` will be a better interim option until we can get that work done. Will implement today.